### ISSUE TITLE
Dynamic Tag missing for Percentage value on Progress Bar

### ISSUE THREAD

**Issue by Psmeg** [elementor/elementor#10835]
The current build of Elementor Pro (2.8.5) no longer has the dynamic tags option to dynamically set the progress bar percentage value.

This has reverted back to the old slider/value input method.

## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by Worldwebscapes** [comment]
I am experiencing the same results. 

First Image. This is an old progress bar that was previously created and populated.

![image](https://user-images.githubusercontent.com/7572038/76525911-1bc98700-6443-11ea-811e-c890f1839b83.png)
The second Image is a new progress bar. There is no dynamic value.

![image](https://user-images.githubusercontent.com/7572038/76526001-3e5ba000-6443-11ea-980e-2a9fc1198580.png)

Third Image Star widget is not affected.
![image](https://user-images.githubusercontent.com/7572038/76526116-6f3bd500-6443-11ea-9faa-32945c88b9c4.png)

**Comment by marceloaof** [comment]
Given the relevance of this feature, it was the focus of your recent blog post: https://elementor.com/blog/dynamic-user-profile-pictures-numbers/
Estimated fix date?

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v2.9.7
Feel free to update

Thanks!
